src/components/referrals/ImportReferralsComponent.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+    <div
+        class="referralsBox"
+        @submit.prevent="addTest">
+        <h3 class="referralsBox__header">Dodaj skierowanie:</h3>
+        <form class="referralsBox__form referralsForm">
+            <div class="inputBox">
+                <label for="doctorData"> Lekarz:</label>
+                <select
+                    id="doctorData"
+                    v-model="nameAndSurnameDoctor">
+                    <option>Zaimportowane z dnia:</option>
+                </select>
+            </div>
+
+            <div class="inputBox">
+                <label for="researchDate"
+                    >Wybierz dzień wystawienia skierowania:</label
+                >
+                <input
+                    v-model="researchDate"
+                    type="date" />
+            </div>
+
+            <div class="inputBox">
+                <label for="uploadTest">Dodaj wyniki:</label>
+                <div
+                    class="icon"
+                    tabindex="0">
+                    <font-awesome-icon icon="fa-solid fa-file-import" />
+                </div>
+                <input
+                    id="uploadTest"
+                    type="file"
+                    @change="handleChange" />
+                <error-info :message="testFileError" />
+            </div>
+
+            <div class="actionsAndInfo">
+                <button-component
+                    :disabled="disableAddTest"
+                    wide
+                    >Zapisz skierowanie</button-component
+                >
+                <bouncing-balls-component :visible="isLoading" />
+                <error-info :message="error" />
+                <span
+                    v-show="successFlag"
+                    class="dataSend"
+                    >Skierowanie zostało przesłane pomyślnie</span
+                >
+            </div>
+        </form>
+    </div>
+</template>
+
+<script setup>
+    import ButtonComponent from '../ButtonComponent.vue';
+    import BouncingBallsComponent from '../BouncingBallsComponent.vue';
+    import ErrorInfo from '../ErrorInfo.vue';
+    import { computed, ref } from 'vue';
+    import ALLOWED_TYPES from '../../utils/allowedTypes.js';
+    import useStorage from '../../composables/getImage.js';
+    import useUserStore from '../../stores/userStore.js';
+    import { timestamp } from '../../firebase/config.js';
+    import useCollection from '../../composables/useCollections.js';
```

Comment:
add space bewteen imports and group of declaring variables. its better for readable ;)